pygmt/datasets/earth_relief.py

```diff
@@ -129,17 +129,17 @@ def load_earth_relief(
     }
     if data_source not in earth_relief_sources:
         raise GMTInvalidInput(
-            f"Invalid earth relief 'data_source' {data_source}, "
-            "valid values are 'igpp', 'gebco', 'gebcosi' and 'synbath'."
+            f"Invalid earth relief data source '{data_source}'. "
+            "Valid values are 'igpp', 'gebco', 'gebcosi' and 'synbath'."
         )
     # Choose earth relief data prefix
     if use_srtm and resolution in land_only_srtm_resolutions:
         if data_source == "igpp":
             dataset_prefix = "srtm_relief_"
         else:
             raise GMTInvalidInput(
-                f"The {data_source} option is not available if 'use_srtm=True'."
-                " Set data_source to 'igpp'."
+                f"'use_srtm=True' doesn't work with data source '{data_source}'."
```

Comment:
```suggestion
                f"Option 'use_srtm=True' doesn't work with data source '{data_source}'."
```
To not start the sentence with a lowercase word. Does option fit here?